include/CL/sycl/accessor.hpp

```diff
@@ -589,14 +590,31 @@ accessMode == access::mode::discard_read_write) && dimensions == 1) */
 
 
   /* Available only when: accessMode == access::mode::atomic && dimensions == 0*/
-  //operator atomic<dataT, access::address_space::global_space> () const;
+  template<int D = dimensions,
+           access::mode M = accessmode,
+           typename = std::enable_if_t<M == access::mode::atomic && D == 0>>
+  operator atomic<dataT, access::address_space::global_space> () const
```

Comment:
Can you please mark these functions as `HIPSYCL_UNIVERSAL_TARGET`, like the other member functions? This causes it to become available both inside kernels and in host code (`__host__ __device__` in CUDA terminology)